lesson_02/dags/dag_load_local_to_gcs.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+import os
+from airflow import DAG
+from airflow.providers.google.cloud.transfers.local_to_gcs import LocalFilesystemToGCSOperator
+
+os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = "../../de2022-stanislav-rezen-4dd944e14701.json"
+
+with DAG(
+    dag_id='dag_load_local_to_gcs',
```

Comment:
I think you should not use "dag_" prefix as all you can wright in Airflow are DAGs